pkg/client/iam/config.go

```diff
@@ -169,7 +170,7 @@ func getOpenIDConfiguration(c http.Client, baseURL string) (*openIDConfiguration
 	url := strings.TrimRight(baseURL, "/") + openidConfigurationPath
 	resp, err := c.Get(url)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("making GET request: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("executing HTTP GET request : %w", err)
```
Does the error reference the url? If not, I'd add it.